staging/src/k8s.io/kubectl/pkg/util/util.go

```diff
@@ -62,7 +61,7 @@ func ParseFileSource(source string) (keyName, filePath string, err error) {
 	numSeparators := strings.Count(source, "=")
 	switch {
 	case numSeparators == 0:
-		return path.Base(filepath.ToSlash(source)), source, nil
```

Comment:
revert this... filepath.ToSlash ensures the path is using /, so path.Base works properly

Reply:
Done